packages/react/src/sprinkles/sprinkles.css.ts

```diff
@@ -35,6 +35,7 @@ const conditions = {
 } as const;
 
 const states = {
+  active: "&:active",
```

Comment:
let's drop the commit with these changes to `sprinkles.css.ts` since they exist in `main` now